src/main/webapp/script.js

```diff
@@ -196,11 +210,23 @@ function displayMatches() {
 function createCardElement(userID) {
   const cardDiv = document.createElement("div");
   fetch('/user-data?id=' + userID).then(response => response.json()).then((userinfo) => {
+    console.log(userinfo);
     if (userinfo === null) {
         return;
     }
     cardDiv.className = "col card m-5 card-container";
-    const profileImage = createImgElement("images/noBlobStoreImage2.jpg");
+    let profileImage;
+    let foundImage = false;
+    for (let i = 0; i < userinfo.blobkeys.length; i++) {
+      if (userinfo.blobkeys[i] != "") {
```

Comment:
Question for my own understanding: is it ever *expected* that the server pass an empty string in blobKeys?

Reply:
yeah- that's the default. When the user has an image in position 2 say, then bobkeys[2] will have an image path. Otherwise, it will be ""

Reply:
Got it, thanks!